tensorboard/plugins/mesh_visualizer/http_api.md

```diff
@@ -0,0 +1,65 @@
+# Mesh plugin HTTP API
+
+The mesh plugin name is `mesh`, so all its routes are under
+`/data/plugin/mesh`.
+
+## `/data/plugin/mesh/tags`
+
+Retrieves an index of tags containing mesh data.
+
+Returns a dictionary mapping from `runName` (quoted string) to
+dictionaries that map a `tagName` (quoted string) to an object
+containing that tag’s `displayName` and `description`, the latter of
+which is a string containing sanitized HTML to be rendered into the DOM.
+Here is an example:
+    {
+        "train_run": {
+            "mesh_color_tensor": {
+                "samples": 1
+            }, 
+            "point_cloud": {
+                "samples": 1
+            }
+        }
+    }
+
+Note that runs without any mesh tags are included as keys with value the empty dictionary.
+
+## `/data/plugin/mesh/meshes?tag=mesh_color_tensor&run=train_run&sample=0`
+
+Retrieves all necessary metadata to render a mesh with particular tag.
+
+Returns list of metadata for each data (tensor) that should be retrieved next. This includes content type (i.e. vertices, faces or colors), shape of the data, scene configuration, wall time etc. Type of the content maps directly to underlying binary data type, i.e. `float32`, `int32` or `uint8`. 
```

Comment:
Please indicate that the mapping between content types and their enum
constant representations is given by a particular proto definition (with
file path or link).

Reply:
Not sure I understand the comment. Content type (i.e. MeshPluginData.ContentType.VERTEX) is defined in the proto, but the mapping to its data type (i.e. float32) is in the code.

Reply:
The example data belo whas `"content_type": 2`, `"content_type": 3`, and
`"content_type": 1`. Presumably these correspond to face, color, and
vertex data, respectively, but I only know this from reading the proto
definition in `mesh_visualizer/plugin_data.proto`.

Reply:
got it